Apps/Sandcastle/gallery/Manually Controlled Animation.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
```

Comment:
Do you have a model of higher quality that can demonstrate the same functionality for this sandcastle?

Reply:
If you do, can you replace this model and squash the commit history? That way, the addition and removal of the old model won't bloat the size of the change history.

Reply:
I don't have one, but I think I could use the existing "cesium man" model, if that would be better.

Reply:
Cesium Man would work, thank you!